src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -42,8 +40,7 @@ public InventoryMock(@Nullable InventoryHolder holder, int size, @NotNull Invent
 		items = new ItemStack[size];
 	}
 
-	public InventoryMock(@Nullable InventoryHolder holder, @NotNull InventoryType type)
-	{
+	public InventoryMock(@Nullable InventoryHolder holder, @NotNull InventoryType type) {
```

Comment:
You broke the formatting here.
```suggestion
	public InventoryMock(@Nullable InventoryHolder holder, @NotNull InventoryType type)
	{
```